style.css

```diff
@@ -0,0 +1,550 @@
+*,
+*::before,
+*::after {
```

Comment:
Could you explain the before and after pseudo classes you've used, and what they're doing?